### ISSUE TITLE
As a grantee pilot user I want to be able to access the knowledge center through project updates environment

### ISSUE THREAD

**Issue by reitermb** [raft-tech/TANF-app#1693]
**Business Value**
As a Grantee Pilot User, 
I want to be able to be able to help myself navigate through TDP
So that I can insure my understanding of the TDP platform and continue to do my job

**Description**
Delivers a design for an in-app knowledge base which aggregates all onboarding/support content. May also contain existing materials like data coding instructions.

4/25
Planning to utilize Project Updates environment & templates to host this in the pilot

**AC:** 
- [ ] The design is consistent with the team’s past decisions, or a change is clearly documented 
- [ ] The design is usable, meaning... 
   - [ ] It uses [USWDS components and follows it’s UX guidance](https://designsystem.digital.gov/components/), or a deviation is clearly documented 
   - [ ] Language is intentional and [plain](https://plainlanguage.gov/guidelines/); placeholders are clearly documented 
   - [ ] It follows [accessibility guidelines](https://accessibility.digital.gov/) (e.g. clear information hierarchy, color is not the only way meaning is communicated, etc.) 
   - [ ] If feedback identifies bigger questions or unknowns, create additional issues to investigate 
- [ ] Relevant user stories are documented.

**Notes**
All other onboarding tickets deliver the actual onboarding content, this just aggregates it. Blocked by other onboarding content tickets as it requires insight into the full set of content. 
Likely knowledge base areas:

- [ ] Creating & managing a login.gov account
- [ ] Logging into TDP, Getting Access
- [ ] Framing TDP functionality (Data Submission/Resubmission, Requirements for Data and How to Submit Complete Resubmissions)
- [ ] TDP Release Notes (Potential way to frame feature overview from homepage & Getting Started email)
- [ ] How To: Ask Questions, Get Support, Give Feedback

May also be a good place to host some content from https://tanfdata.org/

Design will likely be modeled after informational USWDS page templates, e.g. USWDS' own getting started guide https://designsystem.digital.gov/documentation/getting-started/developers/phase-one-install/

**Tasks**
- [ ] Draft information architecture (page hierarchy, order, labeling)
- [ ] Add links to all content to be hosted in knowledge base
- [ ] Share for internal review
- [ ] Revision as needed

**Documentation**
- Draft in hack.md
[existing login.gov ](https://hackmd.io/t1lPMv2vQkCxcDhtPxrbCg)
[new login.gov](https://hackmd.io/I5zu3_jNSZem9Y2cQpb2TA?both) 
**DD**
- [ ] @lfrohlich has reviewed and signed off

**Comment by lilybl1** [comment]
Locally committed changes for getting started sections and side nav proposals

**Comment by lilybl1** [comment]
@reitermb The first 3 pages are ready for code review. Thanks!

**Comment by lfrohlich** [comment]
I love how it looks! Some comments: 

- I'm not sure that we are allowed to use the ACF logo in the toolbar
- should it be "_the_ TANF Data Portal" throughout?
- may not be necessary, but worth considering highlighting the buttons/links in the screenshot
- Getting Started: Would reframe as something like "This knowledge center will guide you through setting up an account to access the TANF Data Portal, submitting data, and managing your account. It will continue to be updated as new features are added."
**Creating a new login.gov account**
- seems like the header should change to reflect which section you're on
- Create your account bullet #3: "Review the Rules of Use and when ready**,** proceed accordingly" (add comma between "ready" and "proceed"); underline "Rules of Use"
- #8: Make consistent "Login to" vs. "log into" I think log into is correct based on googling
- #10: should we say 1 business day instead? (in case they send it on a friday, for example)
- #12 is the second screenshot necessary here? 
**Use existing login.gov account**
- seems like the header should change to reflect which section you're on
- Some same comments as above

**Comment by ADPennington** [comment]
lgtm! some minor notes:

- what are we trying to say here? `With TANF Data Portal (TDP), a new, secure, web based data reporting system we strive to improve the federal reporting experience for TANF grantees and staff.` Seems like we could just say: `our goal for TDP is to improve....`
- +1  lauren's suggestions on account-related pages. re: second screenshot--1653 has more detailed screenshots for data submission. but i like that we are showing the "first look" of these additional screens that the user should see after account approval

**Comment by lilybl1** [comment]
@lfrohlich @ADPennington @stevenino @reitermb I created the comments with checkmarks of what has been committed. 

The following points, we should discuss as a group for consistency across the experience not just specific to Knowledge Center. 

-  I'm not sure that we are allowed to use the ACF logo in the toolbar - there are currently two favicons between Knowledge Center and Prod TDP site - which one are we going with? 
-  should it be "the TANF Data Portal" throughout? - when I look at our prod site, we only ever refer to this as "TANF Data Portal"

**Comment by lfrohlich** [comment]
@ttran-hub a11y comments (originally posted in #1241)


@ADPennington @lilybl1 @reitermb, I was tagged for A11y but no longer see it here. If the Knowledge-center page is in the stage of modifying, please double check the findings below:
- [siteNav](https://reitermb.github.io/Test/knowledge-center/index.html) - may be a false positive, but please check the <ul `class="usa-sidenav">` was flag for List element has direct children that are not allowed inside `<li>` elements
- Getting Started Welcome to  [TANF Data Portal (TDP)](https://reitermb.github.io/Test/knowledge-center/index.html) – First sentence, tag as subheading H4, change H4 tag to P or H2 
- [Creating a New Login.gov Account](https://reitermb.github.io/Test/knowledge-center/create-new-login.html)– subheading 1 to 12 are tags as H4, change tags to H2.  Since the page contains great screen sample and the page is lengthy, can we add a back-to-top navigation link at the end of the page?
- [Use Existing Login.gov Account](https://reitermb.github.io/Test/knowledge-center/existing-login.html) – same comment as creating new login page
- [Submitting Data](https://reitermb.github.io/Test/knowledge-center/January-2022-Update.html) – 404 error, file not found
- [Submissions/Resubmissions 1637](javascript:void(0);) – missing link to the #1637
- [Resubmitting Complete Data Files](https://reitermb.github.io/Test/knowledge-center/complete-resubmissions.html) – “Navigate to the “TANF Cases”…” subheading is tag as H4, change to H3, Since the page contains great screen sample and the page is lengthy, can we add a back-to-top navigation link at the end of the page?
- [Managing Your Account #1653](https://reitermb.github.io/Test/knowledge-center/August-2021-Update.html) – 404 error, file not found

**Comment by lfrohlich** [comment]
How about: 
The TANF Data Portal is a new, secure, web-based data reporting system designed to improve the federal reporting experience for TANF grantees and federal staff. TDP will allow grantees to easily submit accurate data and be confident that they have fulfilled their reporting requirements. This will reduce the burden on all users, improve data quality, inform better policy and programs, and ultimately help low-income families.

**Comment by lilybl1** [comment]
[ ] Getting Started – update intro content 
[ ] Use.gov banner in the header which should they pull the favicon 
[ ] should it be "the TANF Data Portal" throughout? - discussed that "the" to be added in Knowledge Center, but needs to update on the Home Page in future story 
[ ] Getting Started Welcome to [TANF Data Portal (TDP)](https://reitermb.github.io/Test/knowledge-center/index.html) – First sentence, tag as subheading H4, change to P  
[ ] [Creating a New Login.gov Account](https://reitermb.github.io/Test/knowledge-center/create-new-login.html)– subheading 1 to 12 are tags as H4, change tags to H2.  
[ ] [Use Existing Login.gov Account](https://reitermb.github.io/Test/knowledge-center/existing-login.html) – subheading change tags from H4 to H2 

 

[Nice to Have] Back to Top button - Since the page contains great screen sample and the page is lengthy, can we add a back-to-top navigation link at the end of the page? 

[siteNav](https://reitermb.github.io/Test/knowledge-center/index.html) - may be a false positive, but please check the <ul class="usa-sidenav"> was flag for List element has direct children that are not allowed inside <li> elements – miles to review

**Comment by reitermb** [comment]
@ttran-hub best I can tell the sidenav error _is_ a false positive—I'm not sure why it's popping up but I've confirmed that the only elements directly inside of the lists making up the sidenav are other list elements. They contain other HTML elements (links in this case) but that shouldn't be a violation. 

![image](https://user-images.githubusercontent.com/28515874/174841314-a21259c7-ec8f-4546-b19c-4bd6ec4b2aec.png)

**Comment by reitermb** [comment]
Banner adjustments & (an MVP version of) jump to top button have been added! Should all be ready for QASP again

**Comment by lfrohlich** [comment]
> * [Submitting Data](https://reitermb.github.io/Test/knowledge-center/January-2022-Update.html) – 404 error, file not found
> * Submissions/Resubmissions 1637 – missing link to the [[Design] fTANF Partial to Complete resubmission guide #1637](https://github.com/raft-tech/TANF-app/issues/1637)
> * [Resubmitting Complete Data Files](https://reitermb.github.io/Test/knowledge-center/complete-resubmissions.html) – “Navigate to the “TANF Cases”…” subheading is tag as H4, change to H3, Since the page contains great screen sample and the page is lengthy, can we add a back-to-top navigation link at the end of the page?
> * [Managing Your Account #1653](https://reitermb.github.io/Test/knowledge-center/August-2021-Update.html) – 404 error, file not found

FYI @ttran-hub these sections are not yet ready for review

**Comment by ttran-hub** [comment]
LGTM for pages before Submitting Data

**Comment by lfrohlich** [comment]
@lilybl1 @reitermb  
I noticed a few more, mostly minor things. Once these are addressed it should be all done. Let me know if you want to discuss!

Consistency: when referring to a button or link, make it either bold, italics, or in quotes. For example, under creating a new login.gov accounts:

- "Sign in with LOGIN.GOV for grantees" step 1
- Create an account in step 2
- **Rules of Use** in step 3

**Creating a new login.gov account**
- o	Under step 1 add "the" in Visit _the_ TANF Data Portal
- o	step 4 the button just says "Confirm email address" (no "your")
- o	step 8 typo: Log into the TANF Data Por_t_al (missing "t")
- o	step 11 just realized "Await for Access Next Steps" sounds weird. Should it say "Wait for Access _and_ Next Steps"?  
- o	Not addressed: step 12 is the second screenshot (data files) necessary here?

**Use an existing login.gov account**

- o	To be consistent with gerunds, change heading to "Using an existing login.gov account" (instead of "Use")
- o	Suggest rephrasing subtitle as "If you have an existing Login.gov account, follow these steps to get started with the TANF Data Portal (TDP)." As is the action is split
- o	Step 2 there shouldn't be a hyphen in "sign-in"
- o	step 5 Should it say "Wait for Access _and_ Next Steps"?  
- o	Not addressed: step 6 is the second screenshot (data files) necessary here?

**Comment by stevenino** [comment]
Ready for @ttran-hub  @ADPennington to review again. Moving back over to QASP
